pkg/metrics/metrics.go

```diff
@@ -151,3 +188,40 @@ func (mc MachineCollector) listMachines() ([]*mapiv1beta1.Machine, error) {
 func (mc MachineCollector) listMachineSets() ([]*mapiv1beta1.MachineSet, error) {
 	return mc.machineSetLister.MachineSets(mc.namespace).List(labels.Everything())
 }
+
+const (
+	VsphereProvider string = "vsphere"
```

Comment:
I think this should have a capital S to be consistent with other capitalisations through this provider
```suggestion
	VSphereProvider string = "vsphere"
```